ibtk/include/ibtk/CopyToRootSchedule.h

```diff
@@ -121,15 +121,15 @@ class CopyToRootSchedule
      *
      * \return A reference to this object.
      */
-    CopyToRootSchedule& operator=(const CopyToRootSchedule& that);
+    CopyToRootSchedule& operator=(const CopyToRootSchedule& that) = delete;
 
     void commonClassCtor();
 
     const int d_root_proc;
```

Comment:
Is there a reason why some of these get default values but not others?

Reply:
I guess one reason could be that these were set up by a script, and there were not default values in the original implementation...?